addons/account_edi/controller/main.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import http, _
+from odoo.http import request, content_disposition
+
+
+class EdiDocumentDownloadController(http.Controller):
+    @http.route('/web/edi_document/download', type='http', auth='user')
+    def download_edi_documents(self, ids=None):
+        ids = [int(id) for id in ids.split(',')]
+        content = request.env['account.move'].browse(ids)._create_zipped()
+        headers = [
+            ('Content-Type', 'zip'),
+            ('X-Content-Type-Options', 'nosniff'),
+            ('Content-Length', len(content)),
+            ('Content-Disposition', content_disposition(f'{_("edi_documents")}.zip'))
```

Comment:
I don't know. To me it could stay the same in every language. It was asked during the review but I just talked with PO @Olivier-olma and he confirm that the name could stay 'edi_documents.zip' in every language.